pkg/webui/console/api/index.js

```diff
@@ -77,6 +82,11 @@ export default {
       return instance.post(`${appRoot}/api/auth/logout`)
     },
   },
+  oauth: {
+    logout() {
+      return instance.post(`${oAuthBaseUrl}/api/auth/logout`)
```

Comment:
I'm not sure our CORS headers would allow this (and no, we shouldn't change those)